site/docs/v1/tech/apis/identity-providers/index.adoc

```diff
@@ -33,6 +33,17 @@ To learn how to configure these Identity Providers using the FusionAuth UI, go h
 
 The way a link is established between an identity provider and FusionAuth is determined by the `linkingStrategy` for each identity provider. An API is provided to manually link and unlink a user to a 3rd party identity provider. To learn more about managing links between FusionAuth and a 3rd party identity provider, see the link:/docs/v1/tech/apis/identity-providers/links[Link APIs].
 
+=== Login
+
+This endpoint is accessible across all IdP's. Please reference the `complete the login` section of each IdP you are integrating with for more information regarding the request and response object required.
+
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Complete the login with any Identity Provider
+[.endpoint]
+.URI
+--
+[method]#GET# [uri]#/api/identity-provider/login#
+--
+
```

Comment:
This looks to be a partial API doc - is this intentional? I think this is already documented in each IdP respectively.